lib/ui/views/home/home_viewmodel.dart

```diff
@@ -39,13 +39,13 @@ class HomeViewModel extends BaseViewModel {
   bool showUpdatableApps = false;
   List<PatchedApplication> patchedInstalledApps = [];
   List<PatchedApplication> patchedUpdatableApps = [];
-  String _managerVersion = '';
+  String? _latestManagerVersion = '';
 
   Future<void> initialize(BuildContext context) async {
-    _managerVersion = await AboutInfo.getInfo().then(
+    _latestManagerVersion = await AboutInfo.getInfo().then(
```

Comment:
I think you should delete this since assigning this field twice is unnecessary, and it also assigns the current manager version, which is misleading.

Reply:
oh lol i didnt made that part, since its overwritten right on the next line, ill go ahead and delete it